content/guides/cloud/smart-orchestration.md

```diff
@@ -99,48 +97,47 @@ failure will:
 
 </Alert>
 
-### Toggle cancellation of test runs upon first failure
+### Activate Auto Cancellation in Cypress Cloud
 
 <Alert type="success">
 
 <strong class="alert-header"><Icon name="star"></Icon> Premium Cypress Cloud
 Feature</strong>
 
-**Canceling test runs when a test fails** is a _Smart Orchestration_ feature
-available to users with a
-[Business Cypress Cloud plan](https://cypress.io/pricing).
+**Auto Cancellation** is a _Smart Orchestration_ feature available to users with
+a [Business Cypress Cloud plan](https://cypress.io/pricing).
```

Comment:
and same comment as above, about using ROAR to help an "undecided" reader make an informed decision here